app/services/data/insert-task-generate-direct-payments.js

```diff
@@ -0,0 +1,14 @@
+const config = require('../../../knexfile').asyncworker
```

Comment:
Gary D has updated the insert-task-dwp-check to be generate, merge this in and re-use rather than using new modules